cloudsim-plus/src/main/java/org/cloudbus/cloudsim/brokers/DatacenterBroker.java

```diff
@@ -53,7 +53,7 @@
     Function<Vm, Double> DEFAULT_VM_DESTRUCTION_DELAY_FUNCTION = vm -> -1.0;
 
     /**
-     * Specifies that an already submitted cloudlet, that is in the {@link #getCloudletsWaitingList() waiting list},
+     * Specifies that an already submitted cloudlet, that is in the {@link #getCloudletWaitingList() waiting list},
```

Comment:
Change to "Specifies that an already submitted cloudlet, which is in the {@link #getCloudletWaitingList() waiting list},"